tfhe/src/integer/server_key/radix_parallel/tests_cases_comparisons.rs

```diff
@@ -1512,17 +2052,147 @@ fn test_signed_default_scalar_function<SmartFn, ClearF>(
 
         // Sanity decryption checks
         {
-            let a: i128 = cks.decrypt_signed_radix(&ct_0);
+            let a: T = cks.decrypt_signed_radix(&ct_0);
             assert_eq!(a, clear_0);
         }
 
-        let decrypted_result: i128 = cks.decrypt_signed_radix(&encrypted_result);
+        let decrypted_result: T = cks.decrypt_signed_radix(&encrypted_result);
 
         let expected_result = clear_fn(clear_0, clear_1);
         assert_eq!(decrypted_result, expected_result);
     }
 }
 
+#[cfg(not(feature = "__coverage"))]
+fn integer_signed_unchecked_scalar_min_parallelized_i128(
```

Comment:
can it be macroized like below ?

do the functions need the gating (if needed we can make them pub to avoid clippy beingannoying)